spec/controllers/reconcilliations_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "controller_spec_helper"
+
+RSpec.describe ReconcilliationsController do
+  before(:all) { create_users }
+
+  let(:user) { FactoryBot.create(:user) }
+  let(:facility) { FactoryBot.create(:setup_facility) }
+  let(:item) { FactoryBot.create(:item, facility: facility) }
+  let(:account) do
+    FactoryBot.create(:nufs_account, account_users_attributes: [FactoryBot.attributes_for(:account_user, user: user)]).tap do |account|
+      define_open_account item.account, account.account_number
+    end
+  end
+  let(:order_detail) { place_and_complete_item_order(user, facility, account) }
+
+  before :each do
+    order_detail.change_status! OrderStatus.reconciled
+  end
+
+  describe "DELETE to :destroy" do
+    context "for an administrator" do
+      before :each do
+        sign_in @admin
```

Comment:
Can we use `let`s for defining the users rather than the `@` variables created by `before(:all) { create_users }`? It's an older pattern that I think only really makes sense with the `it_should_all` pattern (which is one of those things I'd like to get rid of).

Reply:
For sure, thanks for the context here! 21dff8c57 does this (and gets rid of the `create_users` call entirely)